digits/templates/models/images/classification/custom_network_explanation.html

```diff
@@ -1,36 +1,107 @@
 {# Copyright (c) 2015-2016, NVIDIA CORPORATION.  All rights reserved. #}
 
-
 <h1>Specifying a custom Caffe network</h1>
 
 <p>
-    You should enter a train_val network here. Some cleanup is done for you automatically:
+    Use this field to enter a Caffe network in .prototxt format.
 </p>
 
+<p>
+    Classification are special in a few ways:
+</p>
 <ul>
     <li>
-        The <i>num_output</i> for each <b>InnerProduct</b> layer which is a network output gets set to the number of labels in the chosen dataset.
+        The Deploy network <b>must contain a Softmax layer</b>.
+        This should produce the only network output.
```

Comment:
I think GoogleNet does not abide by this principle. Do the auxiliary classifiers need to be sent to `SilenceLayer`s in the `deploy` network?